src/directives/ResizeHandler.js

```diff
@@ -15,10 +15,10 @@ export default {
 
     window.addEventListener('resize', function () {
       if (checkIsDesktop() && !prevMatchlg) {
-        sidebar.classList.remove('sidebar-hidden')
+        sidebar.classList.remove('va-sidebar--hidden')
```

Comment:
Pretty sure this could be done from component context. I think "Layout" is a correct spot.